src/Hydrators/UserHydrator.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace ProjectManagementApi\Hydrators;
+
+use ProjectManagementApi\DatabaseConnection;
+use ProjectManagementApi\Entities\User;
+
+class UserHydrator
+{
+    public static function getUsersByProjectId(DatabaseConnection $db, int $id): array
+    {
+        $query = $db->prepare(
```

Comment:
i don't quite understand how you've used AND here, i thought it would need to be WHERE instead?

Reply:
done